recipes/dialign2/build.sh

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
Please remove this line.

Reply:
Looks fixed in [3e34523](https://github.com/bioconda/bioconda-recipes/pull/57080/commits/3e34523c5d1fa9c8361fa633acc4572611ee4d34)